### PR TITLE
doc: add create attachedcluster

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

doc of attachedcluster 
see #287 

The example uses a `hack/local-dev-setup.sh` kind cluster, keep consistent with #294 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: "Integrating the AttachedClusters with Fleet"
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/create-attachedcluster.md:2]: How about `Manage AttachedCluster`

Comment (hzxuzhonghu) [reply docs/content/en/docs/fleet-manager/create-attachedcluster.md:2]: It is not appropriate to to use intergrate

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+linkTitle: "Integrating the AttachedClusters with Fleet"
+weight: 15
+description: >
+  Your roadmap to integrating and managing AttachedClusters with Kurator Fleet.
+---
+
+In Kurator, clusters that were not created by Kurator are referred to as `AttachedClusters`.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/create-attachedcluster.md:9]: s/were/are

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+These clusters can be managed by adding them to the Kurator Fleet, expanding the fleet's control to clusters not originally created by Kurator.
+
+This guide will walk you through the process of creating AttachedCluster resources, using two Kind clusters as examples.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/create-attachedcluster.md:12]: kinds of

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+## Prerequisites
+
+### Cluster Operator and kind Cluster
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/create-attachedcluster.md:16]: No sure what does this mean

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+As the AttachedCluster object is controlled by the cluster-operator, you need to first go to [install cluser operator](/docs/setup/install-cluster-operator.md) page to create kind clusters using `hack/local-dev-setup.sh` and install cluster operator.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/create-attachedcluster.md:18]: > the AttachedCluster object is controlled by the cluster-operator ?

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+```console
+kubectl create secret generic kurator-member1 --from-file=kurator-member1.config=/root/.kube/kurator-member1.config
+kubectl create secret generic kurator-member2 --from-file=kurator-member2.config=/root/.kube/kurator-member2.config
+```
+
+Please note, here we have named the secrets as `kurator-member1` and `kurator-member2` respectively, and set the key to save the kubeconfig in the secret as `kurator-member1.config` and `kurator-member1.config` respectively.
+You can modify these two elements according to your needs.
+
+## Creating AttachedCluster Resources
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/create-attachedcluster.md:34]: Create i think usually it does not use -ing in tile

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+Now that we have the prerequisites sorted out, let's move on to creating the AttachedCluster resources.
+
+We'll start by editing the configuration for the AttachedCluster.
+
+Notice that the `name` and `key` here need to be consistent with the secret generated earlier.
+
+Edit attachedCluster config.
+
+```console
+$ vi ./ac-member1.yaml
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/create-attachedcluster.md:45]: You can write equiavalent of this
```
cat <<EOF | kubectl apply -f -
apiVersion: v1
kind: ConfigMap
metadata:
  name: testMap
  namespace: default
data:
  details:
    host: "localhost:${reg_port}"
EOF
```

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+With the configuration ready, we can apply the resources:
+
+```console
+kubectl apply -f ./ac-member1.yaml
+kubectl apply -f ./ac-member2.yaml
+```
+
+## View resource status
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/create-attachedcluster.md:75]: Keep consistent the tile case, i think we can use small case, but some tiles are upper-cased

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md (hunk 4 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/create-attachedcluster.md:16]: change it to "### Cluster Operator "

File: docs/content/en/docs/fleet-manager/create-attachedcluster.md (hunk 5 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/create-attachedcluster.md:18]: yes, currentlly the AttachedCluster object is controlled by the cluster-operator .
although AttachedClusters are primarily used in the context of Fleet, they should be considered equivalent to regular Clusters and managed in the same way by the Cluster Operator

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**